csv-data-management/info.json

```diff
@@ -77,16 +77,16 @@
           "visible": true,
           "editable": true,
           "value": null,
-          "description": "If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single variable",
-          "tooltip":"If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single variable"
+          "description": "If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single recordSet",
+          "tooltip":"If set , will return rows as recordSet in list of 20 batches otherwise complete result in a single recordSet"
         }
       ],
       "open": true
     },
     {
       "title": "Merge and Extract Data from two CSV",
       "operation": "merge_two_csv_and_extract_data",
-      "description": "This action will extract data from CSV based on column names provided ,by merging two CSV file. User can select option to deduplicate rows based on columns",
+      "description": "This action will extract data from CSV based on column names provided ,by merging two CSV file. User can select option to deduplicate values based on column(s) provided",
```

Comment:
Suggestion: This action will merge two CSV files and extract data based on column names provided. Optionally, ....User can

Reply:
reworded  and updated